### ISSUE TITLE
Migration fails on MySQL 8.0.22

### ISSUE THREAD

**Issue by mintbridge** [laravel-arcanist/arcanist#3]
Publishing then running the migrations fails with the following error:

```
SQLSTATE[42000]: Syntax error or access violation: 1101 BLOB, TEXT, GEOMETRY or JSON column 'data' can't have a default value (SQL: create table `wizards` (`id` int unsigned not null auto_increment primary key, `class` varchar(255) not null, `data` json not null default '[]', `created_at` timestamp null, `updated_at` timestamp null) default character set utf8mb4 collate 'utf8mb4_unicode_ci')
```

**Comment by mintbridge** [comment]
removing the default in the migration works fine but not sure what impact that will have later

**Comment by ksassnowski** [comment]
Maybe I can remove the column default and instead add a default value inside the repository class

**Comment by ksassnowski** [comment]
This is fixed in version 0.2.0 (https://github.com/laravel-arcanist/arcanist/releases/tag/0.2.0)
